dynamoDb.sh

```diff
@@ -3,12 +3,34 @@
 
 # AttributeName=Password,AttributeType=S \
 
+# aws dynamodb create-table \
```

Comment:
Can we take these comments out?

Reply:
If there's a table that already exists, this causes an error, so I had it commented out. I'll leave it in and write a comment.